yds/examples/README.md

```diff
@@ -0,0 +1,20 @@
+# Example of how to send Yandex Data Streams messages by raw HTTP
```

Comment:
Example on

Reply:
I'd also get rid of _raw_ adjective. kinesis records <-> HTTP messages

Reply:
Fixed